Assets/MixedRealityToolkit.Tests/PlayModeTests/InputUtilsTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using System.Collections;
+using UnityEditor;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Input;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    // Tests to verify that the ray utilities methods are functioning correctly
+    public class InputUtilsTests
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            PlayModeTestUtilities.Setup();
+            TestUtilities.PlayspaceToOriginLookingForward();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            PlayModeTestUtilities.TearDown();
+        }
+
+        [UnityTest]
+        public IEnumerator HeadGazeRayTest()
+        {
+            yield return null;
+
+            TestContext.Out.WriteLine("Get the head gaze ray");
+            Ray ray = InputUtils.GetHeadGazeRay();
+            Assert.True(ray.origin == Vector3.zero);
+            Assert.True(ray.direction == new Vector3(0.0f, 0.0f, 1.0f));
```

Comment:
nit: why not use Vector3.Forward? and -Vector3.Forward below?

Reply:
Good question, I didn't notice that, it was code from David.